src/pages/login/login.js

```diff
@@ -65,16 +48,11 @@ const Login = () => {
       .email(errorMessages[language].value.email)
       .required(errorMessages[language].value.empty),
     password: Yup.string()
-      .matches(formRegExp.pass, errorMessages[language].value.pass)
+      .matches(formRegExp.password, errorMessages[language].value.pass)
       .required(errorMessages[language].value.empty),
     staySignedIn: Yup.bool()
   });
```

Comment:
Put validation schema in separate file in validators folder

Reply:
fixed!